sdks/java/testing/test-utils/build.gradle

```diff
@@ -30,12 +30,14 @@ dependencies {
   compile project(path: ":sdks:java:core", configuration: "shadow")
   compile library.java.vendored_guava_26_0_jre
   compile library.java.google_cloud_bigquery
-  compile project(":sdks:java:extensions:google-cloud-platform-core")
-
+  compile "com.google.code.gson:gson:2.8.6"
```

Comment:
there should be a blank line between compile and testCompile lines - like there was before